src/Builder/Stage/FluentFactoryTrait.php

```diff
@@ -0,0 +1,792 @@
+<?php
+
+/**
+ * THIS FILE IS AUTO-GENERATED. ANY CHANGES WILL BE LOST!
```

Comment:
If this file is auto-generated, should it be labeled as such in `.gitattributes`?

Reply:
We already have `/src/Builder/Stage/*.php linguist-generated=true`

Reply:
Noted. I'm not sure why GitHub opted not to present that explanation for hiding the file (maybe "large diffs" take precedence).